ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -85,6 +85,14 @@ tests:
       EC2_INSTANCE_TYPE: c5n.metal
     workflow: openshift-microshift-e2e-metal-tests
 - as: microshift-metal-cache
+  cluster: build02
+  run_if_changed: ^(test/bin/ci_phase_iso_build.sh|test/bin/manage_build_cache.sh)
```

Comment:
Dots may be escaped, but given the path and file name + extension I dont think it will ever cause a collision.